website/docusaurus.config.js

```diff
@@ -20,6 +20,9 @@ const commonDocsOptions = {
   remarkPlugins: [require('@react-native-website/remark-snackplayer')],
 };
 
+const isDev = process.env.NODE_ENV === 'development';
```

Comment:
I think that If this is not used, we can remove it.

Reply:
yes 👍 removed